### PR TITLE
Expandable charm content

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- using the `profile` flag view the charm list in a profile
- click on a charm row (not the name) and it should reveal more info

**Comment by huwshimi** [comment]
[Add charm list expanded content](https://trello.com/c/Y0cZMuQV/129-add-charm-list-expanded-content)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8348/

File: jujugui/static/gui/src/app/assets/css/_components.scss [hunk 1]
```diff
@@ -107,6 +107,7 @@
 @import '../../components/popup/popup';
 @import '../../components/post-deployment/post-deployment';
 @import '../../components/profile/profile';
+@import '../../components/profile/expanded-content/expanded-content';
```
Comment (frankban) [inline jujugui/static/gui/src/app/assets/css/_components.scss]: Put this in alphabetical order?

File: jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js [hunk 2]
```diff
@@ -184,49 +129,25 @@ class ProfileBundleList extends React.Component {
...
-                  <a href={bundle.bugUrl}
-                    onClick={this._stopPropagation.bind(this)}
-                    target="_blank">
-                    Bugs
-                  </a>
-                </div>
+            <ProfileExpandedContent
+              changeState={this.props.changeState}
+              entity={bundle}
+              getDiagramURL={this.props.charmstore.getDiagramURL}
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js]: Do we need to bind this? Just asking for double checking.

**Comment by frankban** [comment]
QA ok.

File: jujugui/static/gui/src/app/assets/css/_components.scss (hunk 1 above)
Comment (huwshimi) [reply jujugui/static/gui/src/app/assets/css/_components.scss]: It looks like we use a pattern of sub-dirs after files, so this is at least consistent.

File: jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js (hunk 2 above)
Comment (huwshimi) [reply jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js]: Thanks, it probably should be.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8356/

**Comment by huwshimi** [comment]
Thanks @frankban :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**